### PR TITLE
style(Tooltip): changed tooltip font

### PR DESCRIPTION
**Pull request by Pratik-050**
**Notes for Reviewers**
Changed font for the CustomTooltip component.

This PR fixes #

Screenshot from 2024-12-03 00-04-05

Screenshot from 2024-12-03 00-03-53

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/custom/CustomTooltip/customTooltip.tsx [hunk 1]
```diff
@@ -13,6 +15,10 @@ type CustomTooltipProps = {
   bgColor?: string;
 } & Omit<TooltipProps, 'title' | 'onClick'>;
 
+const StyledFontWrapper = styled(Typography)(() => ({
+  fontFamily: ['Qanelas Soft Regular'].join(',')
+}));
+
```
Comment (amitamrutiya) [inline src/custom/CustomTooltip/customTooltip.tsx:21]: use the typography from the theme. take a theme as a props in the typography

**Review by amitamrutiya — CHANGES_REQUESTED** [review]

File: src/custom/CustomTooltip/customTooltip.tsx (hunk 1 above)
Comment (sudhanshutech) [reply src/custom/CustomTooltip/customTooltip.tsx:21]: yes i already told yesterday pass variant not font family

Comment (Pratik-050) [reply src/custom/CustomTooltip/customTooltip.tsx:21]: Hello @amitamrutiya @sudhanshutech here we can't use this approach:

const StyledFontWrapper = styled(Typography)(({theme}) => ({
  typography: theme.typography.textH2Regular,
}));

cause the styled component is being imported from the mui library, and the theme tokens are also coming from mui Typography api. So they'll take the default font of MUI which is Roboto.
I've already tried that.

**Comment by leecalcote** [comment]
Designs, not Patterns.

**Pull request closed without merging**